s3auth-hosts/src/main/java/com/s3auth/hosts/DefaultDynamo.java

```diff
@@ -56,58 +58,58 @@
  * Abstraction on top of DynamoDB SDK.
  *
  * <p>The class is mutable and thread-safe.
- *
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
- * @since 0.0.1
  * @checkstyle ClassDataAbstractionCoupling (500 lines)
  * @todo #1 Would be nice to migrate to jcabi-dynamo
+ * @since 0.0.1
  */
 @Immutable
 @ToString
-@EqualsAndHashCode(of = { "client", "table" })
+@EqualsAndHashCode(of = {"table" })
 @Loggable(Loggable.INFO)
 final class DefaultDynamo implements Dynamo {
 
     /**
      * Dynamo DB key, URN of a user.
      */
-    public static final String USER = "user.urn";
+    public static final String USER = "user_urn";
```

Comment:
We can't change the name of the Dynamo DB key, since that's how the current database exists. The tests should accommodate what's there, not the other way around.